repose-aggregator/components/filters/body-extractor-to-header-filter/src/test/scala/org/openrepose/filters/bodyextractortoheader/BodyExtractorToHeaderFilterTest.scala

```diff
@@ -235,7 +235,7 @@ class BodyExtractorToHeaderFilterTest extends FunSpec with BeforeAndAfterEach wi
 
       filter.doFilter(servletRequest, servletResponse, filterChain)
 
-      assertFalse(filterChain.getRequest.asInstanceOf[HttpServletRequest].getHeaders(extractedHeader).hasMoreElements)
+      filterChain.getRequest.asInstanceOf[HttpServletRequest].getHeaders(extractedHeader).asScala.toTraversable shouldBe empty
```

Comment:
It was, `toScala` gives you a `TraversableOnce` which the scalatest collection matcher don't work on.